paper_trail.gemspec

```diff
@@ -52,7 +52,7 @@ has been destroyed.
   # PT supports request_store versions for 3 years.
   s.add_dependency "request_store", "~> 1.4"
 
-  s.add_development_dependency "appraisal", "~> 2.4.1"
+  s.add_development_dependency "appraisal", "~> 2.5.0"
```

Comment:
this was necessary because i couldn't run `appraisal` locally - i think [this fix](https://github.com/thoughtbot/appraisal/commit/97079f000840338edde73ded08ed6687e0b5223a) is what got it working again